touchforms/spec/javascripts/entrycontrols_full_spec.js

```diff
@@ -121,17 +121,22 @@ describe('Entries', function() {
 
     it('Should return DateEntry', function() {
         questionJSON.datatype = Formplayer.Const.DATE;
-        questionJSON.answer = '90-09-26';
+        questionJSON.answer = '1990-09-26';
 
         entry = (new Question(questionJSON)).entry;
         expect(entry instanceof DateEntry).toBe(true);
         expect(entry.templateType).toBe('date');
 
-        entry.answer('87-11-19');
+        entry.answer('1987-11-19');
```

Comment:
Assuming that parsing a date with the yy-mm-dd format will correctly parse something in yyyy-mm-dd format based on this test, but is that correct @benrudolph ?